deploy/virtualbox_xenial_gui_creation/instructions.md

```diff
@@ -0,0 +1,36 @@
+# Base box creation instructions
+
+When creating the base box, log in to the desktop immediately so gsettings will work.
```

Comment:
@irahopkinson Failing to do this is what caused the error you ran into.

Reply:
Thanx